salt/serializers/configparser.py

```diff
@@ -21,18 +21,37 @@
 available = True
 
 
+def _check_preservecase_option(**options):
+    '''
+    Checks options passed to the serialize/deserialze function
+    for the 'casesensitive' argument.  If passed as True,
+    the configparser object will preserve case in items
+    '''
+    preserve_case = False
+    if 'preserve_case' in options:
```

Comment:
this is an anti-pattern that we shouldn't use.

use: preserve_case = options.get('preserve_case', False)

Reply:
This is done b/c ``preserve_case`` isn't a valid keyword argument to the ConfigParser object, which options gets passed to when the ConfigParser is instantiated.  It needs to be taken out of the dict, which is why ``pop`` is used.

Reply:
@lomeroe - This comment was before seeing that you do a pop. Then I suggested using the pop directly, in the next comment.

Reply:
yeah, sorry about that, read over that one too fast or something :)